app/src/main/java/org/mozilla/fenix/components/TabCollectionStorage.kt

```diff
@@ -59,6 +60,7 @@ class TabCollectionStorage(
     fun createCollection(title: String, sessions: List<Session>) {
         collectionStorage.createCollection(title, sessions)
         notifyObservers { onCollectionCreated(title, sessions) }
+        Collections.totalCollections.set(getTabCollectionsCount().toLong())
```

Comment:
Why do we do this each time a collection is created or removed rather than on startup? It could be valuable to comment because other people may have the same questions I have.

Reply:
Initially, I had it at startup but changed it  so that we collect more accurate information. But I am thinking about changing it back to startup again given its a startup telemetry ticket?